plum/parametric.py

```diff
@@ -30,8 +30,37 @@
 
 _dispatch = Dispatcher()
 
+class ParametricTypeMeta(TypeMeta):
+    """Parametric Types must be instantiated with indexing."""
 
-class CovariantMeta(TypeMeta):
+    def __getitem__(self, *p):
+        # Type[Par1, Par2] creates a new parametric type
```

Comment:
Would it make sense to move this comment and the below comment to the docstring of the class, so that the generated docs also include them?